update_service/installing-update-service.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-update-service"]
+= Installing and configuring the OpenShift Update Service
+include::modules/common-attributes.adoc[]
+:context: update-service
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+You can install and configure the OpenShift Update Service to provide over-the-air updates for your cluster and its underlying operating system.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-prereqs"]
+== Prerequisites
```

Comment:
For the clusters that are installed in a restricted network, OLM by default cannot access the OperatorHub sources. So the osus operator is not able to be installed directly if OLM is not enabled in restricted network. I would suggest to include the [Using Operator Lifecycle Manager on restricted networks](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html) in the prerequisites.